src/diete.css

```diff
@@ -69,3 +69,8 @@
 	--half-padding: calc(var(--padding) / 2);
 	--border-radius: 0.25rem;
 }
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
```

Comment:
It is not a good idea to redefine global css behavior in the `diete.css`, because it may override other application's preferences. It is better to apply these kind of changes in the component that needs it or keep it in the `app.pcss`.

Furthermore this change breaks the sidebar:
![Screenshot_20240613_114729](https://github.com/snaha-net/diete.design/assets/230163/a0870f47-0435-4b6d-8100-05fd7c621a18)

Reply:
AFIK we actually need this for the diete components (e.g. the checkboxes and radio buttons rely on this) and also for this page. I am not sure how to best approach this, whether we should add it to individual components and remove it on the page or if this should be part of `diete.css`.

BTW I have no idea why moving it from `app.css` to `diete.css` alters the sidebar